app/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@auth0/auth0-react": "^2.2.4",
+    "@types/lodash": "^4.17.13",
+    "@types/mapbox-gl": "^3.4.0",
+    "bootstrap": "^5.3.3",
+    "date-fns": "^4.1.0",
+    "graphql": "^16.9.0",
+    "graphql-request": "^7.1.0",
+    "immer": "^10.1.1",
+    "lodash": "^4.17.21",
+    "mapbox-gl": "^3.7.0",
+    "next": "14.2.13",
+    "react": "^18",
+    "react-bootstrap": "^2.10.4",
+    "react-datepicker": "^7.5.0",
```

Comment:
this datepicker gets the job done, but it looks really dated. i have not found a framework-agnostic alternative but didn't look very hard.

Reply:
ah yes, here we go: https://github.com/gpbl/react-day-picker this one is used by [shadcn/ui](https://ui.shadcn.com/docs/components/calendar).

Reply:
as long as we don't go with the one that I picked for VZV! 🙈